js/index.js

```diff
@@ -0,0 +1,694 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+/**
+ * @fileoverview The main user facing module. Exports WebDriver's primary
+ * public API and provides convenience assessors to certain sub-modules.
+ */
+
+'use strict';
+
+const chrome = require('selenium-webdriver/chrome');
+const edge = require('./edge');
+const firefox = require('selenium-webdriver/firefox');
+const _http = require('selenium-webdriver/http');
+const ie = require('selenium-webdriver/ie');
+const actions = require('selenium-webdriver/lib/actions');
+const by = require('selenium-webdriver/lib/by');
+const capabilities = require('selenium-webdriver/lib/capabilities');
+const command = require('selenium-webdriver/lib/command');
+const error = require('selenium-webdriver/lib/error');
+const events = require('selenium-webdriver/lib/events');
+const input = require('selenium-webdriver/lib/input');
+const logging = require('selenium-webdriver/lib/logging');
+const promise = require('selenium-webdriver/lib/promise');
+const session = require('selenium-webdriver/lib/session');
+const until = require('selenium-webdriver/lib/until');
+const webdriver = require('selenium-webdriver/lib/webdriver');
+const opera = require('selenium-webdriver/opera');
+const phantomjs = require('selenium-webdriver/phantomjs');
+const remote = require('selenium-webdriver/remote');
+const safari = require('selenium-webdriver/safari');
+
+const Browser = capabilities.Browser;
+const Capabilities = capabilities.Capabilities;
+const Capability = capabilities.Capability;
+const Session = session.Session;
+const WebDriver = webdriver.WebDriver;
+
+
```

Comment:
What is the intention with this file? It looks like we're re-exporting the entire Selenium API.

Reply:
removed the index.js file.